Realm/RLMObject.h

```diff
@@ -428,13 +482,41 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ Information about a specific property which changed in an RLMObject change notification.
+ */
+@interface RLMPropertyChange : NSObject
+
+/**
+ The name of the property which changed.
+ */
+@property (nonatomic, readonly, strong) NSString *name;
+
+/**
+ The value of the property before the change occurred. This will always be `nil`
+ if the change happened on the same thread as the notification and for `RLMArray`
+ properties.
+
+ For object properties this will give the object which was previously linked to,
```

Comment:
Does this apply to the Swift API as well? It's not mentioned in the comment for the corresponding Swift property.